outbound/bulk_extractor.md

```diff
@@ -0,0 +1,127 @@
+# **Outbound Reviews: Scanning Tools for PII and Secrets**
+
+**Bulk\_extractor** is a forensic tool used to extract useful information from disk images, files and other digital media. It looks for strings of interest, such as email addresses, URLs, credit card numbers and other artifacts. Unlike traditional forensic tools, **bulk\_extractor** does not mount or parse the file system, making it faster and more suitable for unstructured data. It is commonly used for cybersecurity investigations, data recovery and digital forensics. 
+
+**Gitleaks** is an open-source tool designed to detect and prevent secrets (like API keys, tokens and passwords) from being committed to Git repositories. It scans repositories, including their entire history, for hardcoded secrets and generates reports to help developers address potential security vulnerabilities. **Gitleaks** is specifically tailored to source code repositories, making it highly effective in development environments.
+
+### **Uses:**
+
+| Checks for | GitLeaks | Bulk-Extractor |
+| :---- | :---- | :---- |
+| Emails |  | ✅ |
+| Credit Cards |  | ✅ |
+| File fragments (JPEGs, PDF headers) |  | ✅ |
+| JSON snippets |  | ✅ |
+| Phone numbers |  | ✅ |
+| URLs |  | ✅ |
+| IP addresses |  | ✅ |
+| DNS queries |  | ✅ |
+| Base64-encoded data |  | ✅ |
+| PII |  | ✅ |
+| API keys | ✅ |  |
+| SSH keys | ✅ |  |
+| Hardcoded passwords | ✅ |  |
+| Tokens | ✅ |  |
+| Secrets | ✅ | ✅ |
+
+### **Key Differences:**
+
+Bulk-extractor:
+
+* Analyzes raw data and disk images  
+* Digital forensics and data recovery  
+* Source: Files, disk images, raw data
+
+GitLeaks:
+
+* Scan for Git repo secrets  
+* Source code and secrets  
+* Used for repos
+
+### **Bulk-Extractor Installation for Mac:**
+
+Ensure you are installing bulk\_extractor in your local drive not within the repository or disk you are checking.
+
+* \`brew install autoconf\`  
+* \`brew install automake\`  
+* \`brew install libtool\`  
+* \`brew install pkg-config\`
+
+* \`git clone \--recurse-submodules [https://github.com/simsong/bulk\_extractor.git](https://github.com/simsong/bulk_extractor.git)\`
+
+* \`cd bulk\_extractor\`
+
+* \`./bootstrap.sh\`  
+* \`./configure\`  
+* \`make\`  
+* \`make install\`
+
+### **Bulk-Extractor Installation for Windows**
+
+* Download the Bulk Extractor Windows binary from the [official website](https://github.com/simsong/bulk_extractor).  
+* Extract the downloaded zip file to your preferred directory.  
+* Add the Bulk Extractor binary folder to your system PATH:  
+  * Press Win \+ R, type sysdm.cpl, and hit Enter.  
+  * Navigate to the "Advanced" tab and click on "Environment Variables."  
+  * Under "System Variables," locate and edit the Path variable.  
+  * Add the path to the Bulk Extractor folder and click OK.
+
+		or
```

Comment:
this "or" is rendered as a code block for some reason

Reply:
Fixed.